ttyd/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: ttyd_web_1
+      APP_PORT: 7681
+
+  web:
+    image: tsl0922/ttyd:1.7.3
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      # We should access to all umbrel files from ttyd terminal
+      # - ${UMBREL_ROOT}
```

Comment:
Unfortunately, we can't allow access to the umbrel root directory for security reasons. The app needs to be properly sandboxed. We could mount ${APP_DATA_DIR}/data so that created files, as well as things like `.bashrc` and `.profile`, are persisted.